ccd-definition/SearchResultFields.json

```diff
@@ -1,37 +1,44 @@
 [
+  {
```

Comment:
we don't want this to appear in the search results, please remove

Reply:
Followed screenshots from divorce and added it in results as well. But it is not provided ticket description removed it from results now